ResearchKit/Common/ORKFormStep.h

```diff
@@ -129,6 +129,14 @@ ORK_CLASS_AVAILABLE
 @property (nonatomic, copy, readonly) NSString *identifier;
 
 /**
+ A Boolean value indicating whether the form item is optional.
+ 
+ The default value of this property is `NO`. When the value is `YES`, this form item doesn't need
```

Comment:
I was thinking that defaulting it to `NO` was more coherent with the previous behaviour where all *form items* were non-optional.

But let me know if you prefer it to default to `YES` and I'll change it.